modules/nf-core/maxbin2/main.nf

```diff
@@ -27,7 +28,15 @@ process MAXBIN2 {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def associate_files = reads ? "-reads $reads" : "-abund $abund"
+    def associate_files = ""
```

Comment:
```suggestion
    if () {error("ERROR: MaxBin2 can only accept one of `reads` or `abund`, no both. Check input.")}
    def associate_files = ""
```

Reply:
I am unsure here, is that supposed to be 
`if (reads && abund) {error("ERROR: MaxBin2 can only accept one of `reads` or `abund`, no both. Check input.")}`
?

Reply:
I updated the suggestion @d4straub if you want to commit

Reply:
Thanks!